CMakeLists.txt

```diff
@@ -126,6 +126,22 @@ set(CMAKE_RANLIB ${TOOLCHAIN_PREFIX}ranlib)
 set(CMAKE_STRIP ${TOOLCHAIN_PREFIX}strip)
 set(CMAKE_AR ${TOOLCHAIN_PREFIX}ar)
 
+foreach(lang ASM C CXX OBJC OBJCXX)
+  unset(CMAKE_${lang}_OSX_COMPATIBILITY_VERSION_FLAG)
```

Comment:
Nitpick: the indentation here should be four spaces.

Reply:
sure